### PR TITLE
Switch direction link

### LINKED ISSUE #12: Switch direction
If we are on the page /bth/bri, there should be a hyperlink at the bottom of the page to switch to /bri/bth

### PR DESCRIPTION
**Pull request by AverageMarcus**
Fixes #12 

Added a "Switch Direction" link to the bottom of pages with a to and from station.

### CODE REVIEW

File: nationalrail.js [hunk 1]
```diff
@@ -84,12 +84,14 @@ function getDepartures(stations, callback) {
...
-	
+
 	getDepartureObject(stations, function(departureObject) {
 		console.log(stations);
 		var jadeResponse = {
 			content: generateDepartureHtml(departureObject),
-			pageTitle: 'trntxt: ' + departureObject.fromStation.stationCode
+			pageTitle: 'trntxt: ' + departureObject.fromStation.stationCode,
+			fromStation: departureObject.fromStation.stationCode,
+			toStation: departureObject.toStation.stationCode
```
Comment (danielthepope) [inline nationalrail.js]: Code is failing at L94 because departureStation is undefined

File: resources/template.jade [hunk 2]
```diff
@@ -3,7 +3,12 @@ html
 		include head.jade
 	body
 		p
-			em trntxt.uk 
+			em trntxt.uk
 				small (beta)
 		!= content
+
+		if toStation != undefined and fromStation != undefined
```
Comment (danielthepope) [inline resources/template.jade]: `and` should be `&&`

File: resources/template.jade [hunk 3]
```diff
@@ -3,7 +3,12 @@ html
 		include head.jade
 	body
 		p
-			em trntxt.uk 
+			em trntxt.uk
```
Comment (danielthepope) [inline resources/template.jade]: That space was needed. Should I have used `&nbsp;` instead?

File: resources/template.jade (hunk 2 above)
Comment (AverageMarcus) [inline resources/template.jade]: Thats odd. I used an online Jade test thing and `and` was fine. Jade is horrible!

File: resources/template.jade (hunk 3 above)
Comment (AverageMarcus) [inline resources/template.jade]: Ah sorry, I have trailing spaces auto-removed.

**Pull request merged**